<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Helmholtz Cage GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Magnetic Field (uT):&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_X"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_Y"/>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Z:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="lineEdit_Z"/>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="calibrateBtn">
        <property name="text">
         <string>New Calibration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadCalBtn">
        <property name="text">
         <string>Load Previous Calibration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopBtn">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>283</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" rowspan="5">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="maximumBlockCount">
       <number>200</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Magnetic Field Readings:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="x">
         <property name="text">
          <string>Current X:   </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="y">
         <property name="text">
          <string>Current Y:     </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="z">
         <property name="text">
          <string>Current Z:    </string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="curMag">
         <property name="text">
          <string>Magnitude:     </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="detailsBtn">
      <property name="text">
       <string>Show Details</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>plainTextEdit</zorder>
   <zorder>groupBox</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>detailsBtn</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Connect"/>
    <addaction name="action_Disconnect"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_Configure"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Connect"/>
   <addaction name="action_Disconnect"/>
   <addaction name="action_Configure"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Connect">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/connect.png</normaloff>:/connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to Serial Port</string>
   </property>
  </action>
  <action name="action_Disconnect">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/disconnect.png</normaloff>:/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect From Serial Port</string>
   </property>
  </action>
  <action name="action_Configure">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/settings.png</normaloff>:/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure Srial Port</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/application-exit.png</normaloff>:/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About Program</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="buttons.qrc"/>
 </resources>
 <connections/>
</ui>
